src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1

```diff
@@ -338,18 +314,8 @@ function Test-ValidateLogicApp
 	# Test 2: Using definition object and parameter file.
 	$definition = [IO.File]::ReadAllText($definitionFilePath)
 	Test-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $locationName -Definition $definition -ParameterFilePath $parameterFilePath
-    
-	# Test 3: Failure for an invalid definition.
-	try
-	{
-		Test-AzLogicApp -ResourceGroupName $resourceGroupName -Name $workflowName -Location $locationName -Definition '{}'
```

Comment:
Why are you removing this test?

Reply:
Based on the error message I was getting back, I thought it was a test case that would need to be handled in the parameters itself (like fixing the input of the cmdlet to accept an actual class instead of `Object`). Looking deeper, that doesn't seem to be the case. It's just a bad error message.

I've added the test back in with the new error message